content/docs/tasks/policy-enforcement/denial-and-list/index.md

```diff
@@ -245,7 +245,15 @@ subnet.
 
 ## Cleanup
 
-* Remove the mixer configuration:
+* Remove the mixer configuration for attribute-based white- and blacklisting:
```

Comment:
Could you change all instances of "mixer" with "Mixer" in this doc? Mixer is a proper name...

Thanks.

Reply:
Done! Thanks for catching them.